src/test/storage/storage_manager_test.cpp

```diff
@@ -1,4 +1,5 @@
-﻿#include <memory>
+﻿
```

Comment:
why the empty line?

Reply:
Adding an empty line before the include fixed the two false linting errors - maybe we should have excluded this change in the submission though, as it is not providing any information.